README.md

```diff
@@ -234,6 +200,22 @@ To build a new release package for foreman project for example, do this:
 
     $ tito release koji-foreman
 
+## Releasing a package
+
+After a PR has been merged to add or update a package, then the package should be released.
```

Comment:
In case it was added it's important to add the packages to the right koji tags. In manual mode you can use https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rel-eng/compare-with-tags.rb